src/components/Home/HomeMapList.jsx

```diff
@@ -51,7 +51,21 @@ class HomeMapList extends Component {
                     </span>
                     <p className='truncate'>
                       {post.post_content}
-                      <Link to={'/post/' + post.id}>Read More</Link>
+                    </p>
+                    <p>
+                      <strong>
+                        <Link to={
+                          { 
+                            pathname: '/post/' + post.id,
+                            state:{
+                              post: post,
```

Comment:
Cool thing about ES6 syntax is that you can just simply do:
```
state: {
  post,
  color: this.postColorHandler(post),
}
```

When the key and value names are the same, you don't have to write both down.